docs/graphql/manual/schema/relationships/remote.rst

```diff
@@ -0,0 +1,96 @@
+.. meta::
+   :description: Adding a remote schema relationship with Hasura
+   :keywords: hasura, docs, remote relationship, remote join, remote schema, data federation
+
+.. _add_remote_relationship:
+
+Remote relationships
+====================
+
+Remote joins extend the concept of joining data across tables, to joining across tables *and* remote data sources. Once you create relationships between types from your database and types created from APIs, you can then "join" them by running GraphQL queries.
```

Comment:
We should be consistent with the wording. When we say "Remote relationships" in the title, we should also use it in the rest of the page.